examples/blocksparse_attention/example_tilelang_sparse_gqa_decode_varlen_indice.py

```
@@ -421,6 +422,56 @@ def main(batch=8, heads=32, heads_kv=8, max_cache_seqlen=8192, dim=128, dim_v=12
     print("sparse time: ", (time.time() - start) / 100 * 1000)
 
 
+def benchmark(batch=8, heads=32, heads_kv=8, max_cache_seqlen=8192, dim=128, dim_v=128, sparse_ratio=0.8, block_size=32):
+    batch, heads, heads_kv, max_cache_seqlen, dim, dim_v = batch, heads, heads_kv, max_cache_seqlen, dim, dim_v
+    sparse_ratio = sparse_ratio
+    block_size = block_size
+    max_selected_blocks = int(math.ceil(max_cache_seqlen * (1 - sparse_ratio) / block_size))
```

Ruff (0.14.8)

Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)